### PR TITLE
Fix 178

### PR DESCRIPTION
**Pull request by crtrott**

### CODE REVIEW

**Comment by crtrott** [comment]
Fixes #178

**Review by mhoemmen — APPROVED** [review]
Thanks Christian! : - D

File: include/experimental/__p0009_bits/extents.hpp [hunk 1]
```diff
@@ -142,7 +142,7 @@ class extents
   template <size_t... Idxs>
   MDSPAN_FORCE_INLINE_FUNCTION
   static constexpr
-  index_type _static_extent_impl(size_t n, std::integer_sequence<size_t, Idxs...>) noexcept {
+  std::size_t _static_extent_impl(size_t n, std::integer_sequence<size_t, Idxs...>) noexcept {
     return _MDSPAN_FOLD_PLUS_RIGHT(((Idxs == n) ? Extents : 0), /* + ... + */ 0);
```
Comment (mhoemmen) [inline include/experimental/__p0009_bits/extents.hpp:146]:
```suggestion
    return _MDSPAN_FOLD_PLUS_RIGHT(((Idxs == n) ? Extents : 0), /* + ... + */ std::size_t(0));
```
?

**Review by mhoemmen — APPROVED** [review]

File: include/experimental/__p0009_bits/extents.hpp (hunk 1 above)
Comment (mhoemmen) [reply include/experimental/__p0009_bits/extents.hpp:146]: Discussed; this should not be necessary.

Comment (crtrott) [reply include/experimental/__p0009_bits/extents.hpp:146]: ah I see what you mean, no you don't need that cast their in the end, you do need the extra argument though

**Pull request merged**